docs/tutorials/code/hello-world/task.yaml

```diff
@@ -31,10 +31,10 @@ execute: |
   # [docs:skopeo-copy-end]
 
   rm hello_1.0_amd64.rock
-  docker images
+  docker images hello:1.0
 
   # [docs:docker-run]
-  docker run --rm hello:1.0
+  docker run --rm hello:1.0 exec hello -t
```

Comment:
this one could be a service right?

Reply:
yep it could. It was re-written this way because Pebble doesn't handle well services with a very short lifetime, so if that was a service, the container would keep running after the printout